playbooks/roles/cyclecloud_cluster/projects/common/cluster-init/files/nhc/nhc_common.conf.j2

```diff
@@ -57,7 +57,9 @@
 # * || check_ps_service -S -u root crond
 # * || check_ps_service -r -d qmgr postfix
 # * || check_ps_service -S -u root sshd
+{% if ( monitoring.telegraf | default(true) ) %}
```

Comment:
as the failure is silent it shouldn't be an issue. although it's cleaner to not do it